relayer/processor/metrics.go

```diff
@@ -26,18 +26,18 @@ func (m *PrometheusMetrics) SetLatestHeight(chain string, height int64) {
 	m.LatestHeightGauge.WithLabelValues(chain).Set(float64(height))
 }
 
-func (m *PrometheusMetrics) SetWalletBalance(chain, key, address, denom string, balance float64) {
-	m.WalletBalance.WithLabelValues(chain, key, address, denom).Set(balance)
+func (m *PrometheusMetrics) SetWalletBalance(chain, gasPrice, key, address, denom string, balance float64) {
+	m.WalletBalance.WithLabelValues(chain, gasPrice, key, address, denom).Set(balance)
 }
 
-func (m *PrometheusMetrics) SetFeesSpent(chain, key, address, denom string, amount float64) {
-	m.FeesSpent.WithLabelValues(chain, key, address, denom).Set(amount)
+func (m *PrometheusMetrics) SetFeesSpent(chain, gasPrice, key, address, denom string, amount float64) {
+	m.FeesSpent.WithLabelValues(chain, gasPrice, key, address, denom).Set(amount)
 }
 
 func NewPrometheusMetrics() *PrometheusMetrics {
 	packetLabels := []string{"path", "chain", "channel", "port", "type"}
 	heightLabels := []string{"chain"}
-	walletLabels := []string{"chain", "key", "address", "denom"}
+	walletLabels := []string{"chain", "gasPrice", "key", "address", "denom"}
```

Comment:
Let's use snake_case for label names.

https://prometheus.io/docs/practices/naming/

Also, similar to the comment on trustingPeriod, having a label with many variants like this one promotes creating too many time series.

Reply:
Thanks for the references.
Although, I don't anticipate the gas_price to change; it should not create extra series. This value is being pulled from the relayers config file.